geocat/comp/climatologies.py

```diff
@@ -430,6 +434,7 @@ def calendar_average(
 def climatology_average(
         dset: typing.Union[xr.DataArray, xr.Dataset],
         freq: str,
+        custom_seasons: list = None,
```

Comment:
Whoops sorry missed this one. The typing should match the docstring.
```suggestion
        custom_seasons: typing.Union[list, str] = None,
```

Reply:
Good catch!